Models/coordinates.py

```diff
@@ -0,0 +1,15 @@
+import sys
+from math import sqrt
+
+
+class Coordinates:
+    def __init__(self, latitude, longitude):
+        try:
+            self.latitude = float(latitude)
+            self.longitude = float(longitude)
+        except (ValueError, TypeError) as error:
+            print(f"Invalid Coordinates Provided: {error}")
```

Comment:
Is it ok to have hardcoded values in classes or there is something else we can do with them?

Reply:
Hardcoded values are bad, but i don't really understand about what values do you talk here? If it's about my force cast to float i can see why it's not ok to have it here.  If you are talking about the print message i guess i could make a custom exception that has a custom message.

Reply:
Nice! I was referring to both and I really like your idea about having a custom exception with a message :100: